### PR TITLE
Vaccination Server Test (EXPOSUREAPP-6890)

### PR DESCRIPTION
**Pull request by chris-cwa**
Unit tests for vaccination server.

### CODE REVIEW

**Review by mtwalli — DISMISSED** [review]
Lgtm

File: Corona-Warn-App/src/test/java/de/rki/coronawarnapp/vaccination/core/server/valueset/VaccinationServerTest.kt [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+    /**
+     * contains both binary and signature
+     */
+    private val exportZip = File("src/test/resources/export.zip")
+
+    /**
+     * binary is missing
+     */
+    private val invalidExportZip = File("src/test/resources/export_invalid.zip")
```
Comment (d4rken) [inline Corona-Warn-App/src/test/java/de/rki/coronawarnapp/vaccination/core/server/valueset/VaccinationServerTest.kt:32]: The names are a bit generic.
Maybe `resources/vaccination/valueset_(default|invalid).zip`?

**Review by d4rken — COMMENTED** [review]
Just the one comment on the filenames, otherwise nice test.

**Review by BMItr — DISMISSED** [review]
if you want change the filename of the export.bin (as d4ken mentioned)
otherwise lgtm!

File: Corona-Warn-App/src/test/java/de/rki/coronawarnapp/vaccination/core/server/valueset/VaccinationServerTest.kt (hunk 1 above)
Comment (chris-cwa) [reply Corona-Warn-App/src/test/java/de/rki/coronawarnapp/vaccination/core/server/valueset/VaccinationServerTest.kt]: renamed it

**Review by d4rken — APPROVED** [review]

**Review by mtwalli — APPROVED** [review]

**Pull request merged**